### DISCUSSION TITLE
search dialog box intrusive

### DISCUSSION THREAD

**Discussion by TobyHewing** [dbeaver/dbeaver#35763] [Q&A] [answered]
### Description

When wanting to find / replace in a script file pressing command - F would pop up a very small search box in the top right of the script window. 2 rows, 1 for find and 1 replace where I could still specify 'in selection', 'use regular expression' etc but since the last couple of versions all I seem to get is the large search dialog box that pops up in the middle of the screen and is very intrusive. I don't recall changing any setting to cause this behaviour, can I revert to the previous small find / replace?

### DBeaver Version

Community Edition Version 24.2.0.202408152322

### Operating System

macOS Sonoma 14.7 (23H124)

**Comment by E1izabeth** [comment]
Hi, @TobyHewing 
This is an updated UI of the Find/Replace dialog. It works the same way.
This dialog comes with the Eclipse platform, on which DBeaver relies. We can't change it.

**Comment by ErnieZapata** [comment, marked as answer]
I also find the new Find/Replace overlay annoying but found this - https://stackoverflow.com/questions/79012836/eclipse-2024-09-is-there-a-way-to-get-the-old-find-replace-window-back

**Comment by E1izabeth** [comment]
Thank you for sharing, @ErnieZapata
